### ISSUE TITLE
Why the feature dimension in WikipediaNetwork dataset inconsistent with the origin paper?

### ISSUE THREAD

**Issue by Xzh0u** [pyg-team/pytorch_geometric#3896]
### 🐛 Describe the bug

I found that the feature dimension in WikipediaNetwork dataset is inconsistent with the origin paper.
The WikipediaNetwork dataset provided by WikipediaNetwork class in torch_geometric.datasets has the feature in the size of [2277, 128] (https://graphmining.ai/datasets/ptg/wiki/), which means each node has a [1, 128] feature. However, in the origin paper provided by your doc, the dimension of the feature should be far more larger as shown in the picture below.
![image](https://user-images.githubusercontent.com/46866598/150297653-9c9406f2-94b1-4c31-9dd3-23144be2d38c.png)

And your docs describes that node features represent several informative nouns in the Wikipedia pages. But the length of a dictionary should usually far more larger that 128. So is the node feature in the data you provided really means the nouns in the Wikipedia pages?

Thank you! Looking forward to your reply as soon as possible.

### Environment

* PyG version: 1.7.2
* PyTorch version: 1.7.0
* OS: Ubuntu 20.04
* Python version: 3.8
* CUDA/cuDNN version: 10.2

**Comment by rusty1s** [comment]
Yeah, it looks like the features of a node are provided *after* node embedding. Pinging @benedekrozemberczki here to hopefully be able to share some further insights :)

**Comment by Xzh0u** [comment]
@benedekrozemberczki Hello, could you please explain about the features in WikipediaNetwork dataset provided by pyg? What's more, where is the source of the data you provided? Will you later provide a dataset with the same feature dimension as the origin paper?

**Comment by benedekrozemberczki** [comment]
These are the same features after dimensionality reduction with an autoencoder.

**Comment by Xzh0u** [comment]
Thanks for the explaination. May I ask why didn't you use the origin features but to use features after dimensionality reduction with an autoencoder?
